pkgs/tools/graphics/vkbasalt/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub
+, glslang, meson, ninja, pkg-config
+, libX11
+}:
+
+stdenv.mkDerivation rec {
+  pname = "vkbasalt";
```

Comment:
I'm unsure whether `vkBasalt` should be used instead of `vkbasalt` in this commit.

Reply:
Uppercase letters are fine here, even in the attrnames these days.